CIPs/cip-0032.md

```diff
@@ -0,0 +1,120 @@
+---
+cip: 32
+title: Attestation Incentives
+author: Cody Born <cody@clabs.co>
+discussions-to: https://github.com/celo-org/celo-proposals/issues/144
+status: Draft
+type: Standards
+category: Ring 1
```

Comment:
@prestwich I believe this is accurate, but wanted to double check. It adds slashing via a core contract if a validator's attestation service isn't up, but it doesn't necessarily impact Celo blockchain's functionality (only impacts Valora's given its reliance on the attestation service).

Reply:
👍